mod/wikirate/set/right/general_overview.rb

```diff
@@ -3,28 +3,38 @@
     handle_edit_general_overview(args) { super args }
   end
 
-  view :content do |args|
+  view :core do |args|
```

Comment:
these views will show a editor even the card is not in edit mode.

While citation button is clicked, we only need to show editor with tips to copy of the general overview section. Other things in the company page should be shown as normal. 

missing and core view should be overridden for this case.